server/metrics/metrics.go

```diff
@@ -1418,6 +1421,15 @@ var (
 		ClickhouseStatusLabel,
 	})
 
+	BytesCompressed = promauto.NewCounterVec(prometheus.CounterOpts{
```

Comment:
nit: I think it'd be clearer to have a separate metric for `BytesDecompressed`

Reply:
Done